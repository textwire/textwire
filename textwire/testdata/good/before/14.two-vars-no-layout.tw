<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>{{ title }}</title>
</head>
<body>
    <h2>This @if(visible)is@end nice</h2>
    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nesciunt, vero</p>
</body>
</html>