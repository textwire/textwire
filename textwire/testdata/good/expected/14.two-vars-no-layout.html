<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Textwire is Awesome</title>
</head>
<body>
    <h2>This is nice</h2>
    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nesciunt, vero</p>
</body>
</html>